packages/@okta/vuepress-site/docs/guides/migrate-to-idx/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Migrate to Okta Identity Engine (IDX)
+meta:
+  - name: description
+    content: Our guide shows how to migrate your organization and clients to IDX 
+layout: Guides
+---
+
+## Enable IDX for your organization
+
+# TODO - how to enable IDX?
+
+## Web clients
+
+### Okta-hosted signin
+
+For most authentication flows that involve redirecting to Okta, there should be no other changes needed. Once the feature is enabled, IDX will be used automatically, by default.
+
+#### Customized signin page / custom domain
+
+For most users of the [custom domain](/docs/guides/custom-url-domain/overview/) feature, there are no other changes needed. The default template will detect and use IDX automatically.
+
+However if you have [modified the template](https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/) in certain ways, such as to perform redirects or set cookies, these modificiations may not be compatible with IDX.
```

Comment:
Well, we know that calling `setCookieAndRedirect` will not work. But I'm sure there are other ways it could break. Basically any dependency on authn-based flow or Okta session cookie (sessionToken, session/me, etc.).

Reply:
added some more specifics here in a bulleted list of possible pitfalls. we can add more to the list as we discover them